roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -9,17 +9,20 @@ ansible_ssh_common_args: "{% if 'bastion' in groups['all'] %} -o ProxyCommand='s
 
 kube_api_anonymous_auth: false
 
+# default tlsBootstrapToken
+kubeadm_tlsbootstraptoken: abcdef.0123456789abcdef
+
 # Default value, but will be set to true automatically if detected
 is_atomic: false
 
 # optional disable the swap
 disable_swap: true
 
 ## Change this to use another Kubernetes version, e.g. a current beta release
-kube_version: v1.10.4
+kube_version: v1.11.2
 
 ## Kube Proxy mode One of ['iptables','ipvs']
-kube_proxy_mode: iptables
+kube_proxy_mode: ipvs
```

Comment:
This should be done in a separate PR. It might break user environments if we change the default.